woocommerce/woocommerce-ios
# [Custom Amounts M1] Prepare for release

Closes: #10883


## Description

With this PR we prepare the first milestone of the Custom Amounts project for release by:

- Removing the feature flag and related code
- Removing the legacy code related to adding a single fee to the order
- Updating UI fees references to show "Custom amounts" instead
- Updating Unit Tests
- Updating UI Tests (In progress, shouldn't affect this PR review)

## Testing instructions


Please make sure that the custom amounts functionality works on Release. Also, see that we don't have references to "fees" neither in order creation or order details:

https://github.com/woocommerce/woocommerce-ios/assets/1864060/66343775-686b-40fb-a4f8-2cebd6d94245

---
